_opensearch/install/quickstart.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: default
+title: Quickstart guide
+parent: Install OpenSearch
+nav_order: 1
+---
+
+# Quickstart guide
+
+Get started using OpenSearch and OpenSearch Dashboards by deploying your containers with [Docker](https://www.docker.com/). Before proceeding, you need to [get Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://github.com/docker/compose) installed on your local machine. 
+
+The Docker Compose commands used in this guide are written with a hyphen (for example, `docker-compose`). If you installed Docker Desktop on your machine, which automatically installs a bundled version of Docker Compose, then you should remove the hyphen. For example, change `docker-compose` to `docker compose`.
+{: .note}
+
+## Starting your cluster
+
+You'll need a special file, called a Compose file, that Docker Compose uses to define and create the containers in your cluster. The OpenSearch Project provides a sample Compose file that you can use to get started. Learn more about working with Compose files by reviewing the official [Compose specification](https://docs.docker.com/compose/compose-file/).
+
+1. Before running OpenSearch on your machine, you should review and apply these [important system settings]({{site.url}}{{site.baseurl}}/opensearch/install/important-settings/).
+    - Disable memory paging and swapping performance on the host to improve performance.
+        ```bash
+        sudo swapoff -a
+        ```
+    - Increase the number of memory maps available to OpenSearch.
+        ```bash
+        # Edit the sysctl config file.
+        sudo vi /etc/sysctl.conf
+
+        # Define the max map count.
+        vm.max_map_count=262144
+
+        # Reload the kernel parameters.
+        sudo sysctl -p
+        ```  
+1. Download the sample Compose file to your host. You can download the file with command line utilities like `curl` and `wget`, or you can manually copy [docker-compose.yml](https://github.com/opensearch-project/documentation-website/tree/{{site.opensearch_version}}/assets/examples/docker-compose.yml) from the OpenSearch Project documentation-website repository using a web browser.
+    ```bash
+    # Using cURL:
+    curl -O https://github.com/opensearch-project/documentation-website/tree/{{site.opensearch_version}}/assets/examples/docker-compose.yml
+
+    # Using wget:
+    wget https://github.com/opensearch-project/documentation-website/tree/{{site.opensearch_version}}/assets/examples/docker-compose.yml
+    ```
+1. In your terminal application, navigate to the directory containing the `docker-compose.yml` file you just downloaded, and run the following command to create and start the cluster as a background process.
+    ```bash
+    docker-compose up -d
+    ```
+1. Confirm that the containers are running.
```

Comment:
We can give more details around what to look in the output of this command to confirm containers are running.

Reply:
Ok I'll add a sample output from that command. Thank you!